cmd/nginx-ingress/main.go

```diff
@@ -76,12 +76,13 @@ var (
 	the Ingress Controller will fail to start.
 	The Ingress controller only processes resources that belong to its class - i.e. have the "ingressClassName" field resource equal to the class.
 
-	For Kubernetes < 1.18, the Ingress Controller only processes resources that belong to its class
-	- i.e have the annotation "kubernetes.io/ingress.class" equal to the class.
+	For Kubernetes < 1.18, the Ingress Controller only processes resources that belong to its class - 
+	i.e have the annotation "kubernetes.io/ingress.class" (for Ingress resources)
+	or field "ingressClassName" (for VirtualServer/VirtualServerRoute/TransportServer resources) equal to the class.
 	Additionally, the Ingress Controller processes resources that do not have the class set,
 	which can be disabled by setting the "-use-ingress-class-only" flag
 
-	The Ingress Controller processes all the VirtualServer/VirtualServerRoute resources that do not have the "ingressClassName" field for all versions of kubernetes.`)
+	The Ingress Controller processes all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the "ingressClassName" field for all versions of kubernetes.`)
```

Comment:
Can you clarify this sorry, it processes the resources that *do not* have ingressClassName ?

Reply:
@soneillf5 
 "The Ingress Controller processes all the VirtualServer/VirtualServerRoute/TransportServer resources with an empty "ingressClassName" field for all versions of kubernetes."  ?